src/zzz_od/gui/view/world_patrol/route_operation_editor_dialog.py

```
@@ -143,24 +155,27 @@ def _on_item_changed(self, item: QTableWidgetItem):
+
     def _on_delete_clicked(self):
         """删除选中的操作"""
         current_row = self.table.currentRow()
         if 0 <= current_row < len(self.current_op_list):
-            reply = QMessageBox.question(self, '确认删除', 
-                                       f'确定要删除第{current_row}个操作吗？',
-                                       QMessageBox.StandardButton.Yes | QMessageBox.StandardButton.No)
+            reply = QMessageBox.question(
+                self,
+                gt('确认删除'),
+                f'{gt("确定要删除第")}{current_row}{gt("个操作吗？")}',
```

Ruff (0.15.13)

[warning] String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)